nncf/common/graph/graph.py

```diff
@@ -11,7 +11,21 @@
 import pathlib
 from collections import defaultdict
 from copy import deepcopy
-from typing import Any, Callable, Dict, Generator, KeysView, List, Optional, Tuple, Type, Union, ValuesView, cast
+from typing import (
```

Comment:
?

Reply:
combination of length of line 120 and isort `single_line_exclusions = "typing"`, if you are talking about formatting